addons/website_autocomplete/data/data.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record id="action_autocomplete_provider" model="ir.actions.act_window">
+            <field name="name">Autocompletion Providers</field>
+            <field name="res_model">autocompletion.provider</field>
+            <field name="view_mode">tree,form</field>
+        </record>
```

Comment:
Action is unused so this whole file can be discarded.

Also FYI with this `view_mode` you would also need to declare a list and form view for the model.
Otherwise the ORM will auto-generate the views for us which is not great.